lib/components/user/back-link.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Button } from 'react-bootstrap'
+import styled from 'styled-components'
+
+import Icon from '../narrative/icon'
+
+const Container = styled.div``
+
+const StyledButton = styled(Button)`
+  padding: 0;
+`
+
+const navigateBack = () => window.history.back()
+
+/**
+ * Back link that navigates to the previous location in browser history.
+ */
+const BackLink = () => (
+  <Container>
+    <StyledButton
+      bsStyle='link'
+      onClick={navigateBack}
+    >
+      <Icon name='arrow-left' />  Back
```

Comment:
Given that we're reusing this in a few places, I should point out that I prefer the Icon to have a right margin, rather than separating it from the button text with a space. The reason being that when you hover over the button, there is a portion of whitespace between the icon and text that is underlined. Very minor pet peeve, but a nice detail to fix.